test/user.test.js

```diff
@@ -0,0 +1,245 @@
+const request = require('supertest');
+const expect = require('chai').expect;
+
+const app = require('../index');
+const fixture = require('../fixtures/user');
+
+describe('User controller', () => {
```

Comment:
It is not a unit test, so you don't the the controller insolation, but the whole app. So please change the description to 'CRUD User' or something like this.